restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -169,15 +170,18 @@ public QueryBuilder builder() {
   public boolean maybeCreateTable(String keyspaceName, String tableName) {
     Keyspace ks = dataStore.schema().keyspace(keyspaceName);
 
-    if (ks == null)
-      throw new DocumentAPIRequestException(
-          String.format("Unknown namespace %s, you must create it first.", keyspaceName));
+    if (ks == null) {
+      String message =
+          String.format("Unknown namespace %s, you must create it first.", keyspaceName);
+      throw new ErrorCodeRuntimeException(ErrorCode.DATASTORE_KEYSPACE_DOES_NOT_EXIST, message);
+    }
 
     if (!tableName.matches("^[a-zA-Z0-9_]+$")) {
-      throw new DocumentAPIRequestException(
+      String message =
           String.format(
               "Could not create collection %s, it has invalid characters. Valid characters are alphanumeric and underscores.",
-              tableName));
+              tableName);
+      throw new ErrorCodeRuntimeException(ErrorCode.DATASTORE_TABLE_NAME_INVALID, message);
```

Comment:
Can we avoid duplicating the error message in the exception (line 182) and in the error code enum (`DATASTORE_TABLE_NAME_INVALID`)?

Reply:
Well, if we want to have a non-static messages that can depend on the user params for example, then I don't see a way to do it.. I think it's quite valuable sometimes that you specify the more details in the error message than just a static one? 

Any idea on your mind?

Reply:
You missed the difference I guess, in the above example it depends directly on the collection name (table) provided by the user.